index.js

```diff
@@ -0,0 +1,39 @@
+const validateLineup = (lineup) => {
+  function getSalaries(lineup) {
+    return lineup.salary
+  }
+  function getSalarySum(acc, n) {
+    return acc + n
+  }
+  function getTeamId(lineup) {
+    return lineup.teamId
+  }
+  function getGameId(lineup) {
+    return lineup.gameId
+  }
+  function getPositions(lineup) {
+    return lineup.position
+  }
+
+  const salaryList = lineup.map(getSalaries)
+  const salarySum = salaryList.reduce(getSalarySum, 0)
+
+  const teamIdList = lineup.map(getTeamId)
+  const findTeamDuplicates = teamIdList.filter((item, index) => teamIdList.indexOf(item) != index)
+
+  const gameIdList = lineup.map(getGameId)
+  const findGameDupliactes = gameIdList.filter((item, index) => gameIdList.indexOf(item) != index)
+
+  const positionList = lineup.map(getPositions)
+  const findPositionDuplicates = positionList.filter((item, index) => positionList.indexOf(item) != index)
```

Comment:
This is too abstract to actually work. The only position that can have duplicates is the 'OF' position. But if I had a lineup with the following positions `['P','C','C','1B','2B','3B','SS','OF','OF']` You'd end up with a `findPositionDuplicates` of `['C','OF']` which would pass your check below even though it is an invalid lineup.